### ISSUE TITLE
Implement support for TP measurements

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/jitutils#15]
Since there is already support for crossgenning a set of assemblies, it should be straightforward to measure the time it takes for the crossgen.

**Comment by russellhadley** [comment]
I'm assuming that you mean TP with the IO turned off and just the crossgen time?

**Comment by AndyAyersMS** [comment]
Yes, release builds, no disassembly, possibly with iteration and variance estimation.
